src/mem3/src/mem3_shards.erl

```diff
@@ -36,7 +37,10 @@
 -define(DBS, mem3_dbs).
 -define(SHARDS, mem3_shards).
 -define(ATIMES, mem3_atimes).
+-define(OPENERS, mem3_openers).
 -define(RELISTEN_DELAY, 5000).
+-define(WRITE_TIMEOUT, 1000).
+-define(WRITE_IDLE_TIMEOUT, 30000).
```

Comment:
I've been pondering that as well. I'll make them both configurable since that's easy enough.